### ISSUE TITLE
Question of dense block's kernel size 

### ISSUE THREAD

**Issue by ashurasunny** [vqdang/hover_net#15]
In the paper, we can see that the kernel size of the second convolutional layer in dense block is 3 but in the code it's 5. I just wonder is it a mistake or it will be better.

**Comment by simongraham** [comment]
Hi @ashurasunny 

Thanks for pointing this out. All along, we used a kernel size of 5x5 within the decoder of our network. Therefore, we have made a small mistake when creating the diagram of the network architecture. We will fix this and reupload the paper on ArXiv and ResearchGate.

**Comment by ashurasunny** [comment]
Hi @simongraham , I am really grateful for your reply. BTW,  I think I should pre-process the CoNSeP datase and make the x-y map myself before training the HoVerNet, right? Because the label of original data only have two channels, the first is the mask of each nuclei and the second is the type of each nuclei. I just confirm about this because the information of original data is enough for me to make a dataset for training the hovernet.

**Comment by vqdang** [comment]
Hi @ashurasunny, you don't need to compute anything, you only need to perform the patch extraction for the training set and validation set. The HV (current name of the previous XY-map) maps are calculated on the flight as augmentation during training.

Also, you should create different issue for running/etc. for easier tracking.

**Comment by ashurasunny** [comment]
Hi, @vqdang , thank you for your reply.  I will try it. Actually I tried to implement HoVerNet with Pytorch. It's really helpful of your answer :)

**Comment by simongraham** [comment]
For the generation of horizontal and vertical maps, please refer to issue #7 . Specifically, refer to:

https://github.com/vqdang/hover_net/blob/master/src/loader/augs.py#L192

The pipeline just requires the RGB and the instance label (also the class labels if nuclear type classification is performed) and the horizontal and vertical maps will be created on the fly. If you encounter any problems, please reopen issue #7 .

**Comment by ashurasunny** [comment]
Okay,  thank you very much~
